Src/AutoFoqUnitTest/FoqMethodQueryTest.fs

```diff
@@ -87,6 +105,17 @@ let SelectMethodsDefineCorrectParameters (request: Type) =
                        (resultParameters |> Seq.toArray))) @>
     // Teardown
 
+[<Fact>]
+let ``Builder is correct`` () =
+    let expected =
+        { new ISpecimenBuilder with
+              member this.Create(r, c) = obj() }
+    let sut = FoqMethodQuery expected
+
+    let actual = sut.Builder
+
+    verify <@ expected = actual @>

```

Comment:
Actually, I've just noticed that we're using an old version of Unquote, and these old operators are deprecated in newer versions, [as shown in the Unquote source code](http://git.io/v8KPF):

```
///These '?' suffixed operators conflict with F# 3.0's nullable operators and
/// have been replaced by equivalent '!' suffixed operators.

[<System.Obsolete(
    "These '?' suffixed operators conflict with F# 3.0's nullable operators and
     have been replaced by equivalent '!' suffixed operators.")>]
module Obsolete =
    let inline (=?) x y = test <@ x = y @>
    // ...

let inline (=!) x y = test <@ x = y @>
//...
```

So I'll leave this one as-is, but I'll change an other test added in this PR so it doesn't use that deprecated `=?` operator.

**Tl;dr**

We should consider upgrading Unquote, in a separate Pull Request.